braindecode/models/base.py

```diff
@@ -158,6 +160,86 @@ def sfreq(self):
             )
         return self._sfreq
 
+    @property
+    def input_shape(self) -> Tuple[int]:
+        """Input data shape."""
+        return (1, self.n_chans, self.n_times)
+
+    @property
+    def output_shape(self) -> Tuple[int]:
```

Comment:
Should it be a property or a regular method? Because it does some computations and changes when you add layers

Reply:
this could be negotiated I guess :joy:  -> changing when adding layers does not convince me but the computations needed yes :wink: I'm gonna change that